### PR TITLE
Qingwei/better spot indicator in contract

### PR DESCRIPTION
**Pull request by qingweibinary**

### CODE REVIEW

**Comment by qingweibinary** [comment]
Do not merge

File: src/config/updateChart.js [hunk 1]
```diff
@@ -57,7 +57,12 @@ export default (chart, prevProps, nextProps) => {
 
     if (ticksDiffer) {
         updateTicks(chart, nextProps, mergedContract);
+        const before = chart.xAxis[0].getExtremes().min;
         chart.redraw();
```
Comment (qingweibinary) [inline src/config/updateChart.js]: @borisyankov here is the funny part, the `min` value got changed after `redraw` is invoked

Do you have clue? for example, some plugin that monkey patched the `redraw` method?

Comment (borisyankov) [inline src/config/updateChart.js]: The plugins do intercept the redraw, but don't think they change the min.

**Comment by qingweibinary** [comment]
OK to merge, it fixes some of the problem, but not all, the way we implement start later contract (Null data) is not handle correctly by highchart, I've filed an issue

https://github.com/highcharts/highcharts/issues/5591

**Pull request merged**